src/nvim/globals.h

```diff
@@ -97,8 +97,8 @@ EXTERN struct nvim_stats_s {
 // up).
 #define DFLT_COLS       80              // default value for 'columns'
 #define DFLT_ROWS       24              // default value for 'lines'
-EXTERN int Rows INIT(= DFLT_ROWS);     // nr of rows in the screen
-EXTERN int Columns INIT(= DFLT_COLS);  // nr of columns in the screen
+EXTERN linenr_T Rows INIT(= DFLT_ROWS);     // nr of rows in the screen
+EXTERN colnr_T Columns INIT(= DFLT_COLS);  // nr of columns in the screen
 
```

Comment:
???

Reply:
`linenr_T` is a type to describe number of rows. `Rows` is a variable describing the number of rows.

`colnr_T` is a type to describe number of columns. `Columns` is a variable describing the number of columns.

Reply:
No. `linenr_T` and `colnr_T` and used to describe the number of text columns, not screen columns.

Reply:
If you say so. I don't think that's obvious. I've reverted it.